WooCommerce/Classes/ViewRelated/ReusableViews/LabeledTextViewTableViewCell.swift

```diff
@@ -55,7 +55,16 @@ private extension LabeledTextViewTableViewCell {
     }
 
     func configureProductStatusLabel(productStatus: ProductStatus) {
-        productStatusLabel.text = productStatus.description
+
+        switch productStatus {
+        case .privateStatus:
+            productStatusLabel.text = NSLocalizedString("productDetail.statusLabel",
```

Comment:
non-blocking: should this key be more specific? e.g "productDetail.privateStatusLabel"

Reply:
Absolutely! Fixed here: c290ff788ece2190d5e333dd819491459ee33c10